### ISSUE TITLE
Compile errors Unity 5.5.0b3

### ISSUE THREAD

**Issue by schovan** [Brackeys/MultiplayerFPS-Tutorial#4]
When I open the project in Unity Unity 5.5.0b3, I get compile errors - see attached screenshot.
I don't know how to fix it, please help.
![errors](https://cloud.githubusercontent.com/assets/16373322/18794369/8379c514-81bf-11e6-8e18-36e9386e21ff.png)

**Comment by TheModdersDen** [comment]
I appear to be having the same problem

**Comment by Sctww** [comment]
This has something to do with thew Database Control Asset. Did you download it from the store?
You can find the link [here](https://www.assetstore.unity3d.com/en/#!/content/41337).

**Comment by TheModdersDen** [comment]
Yes. I've tried with both the one that came with this project, and the one from the asset store I believe the problem is that the project is targeting the .NET framework 4.0, and the DatabaseControl.dll asset's assembly info is having it set to target v3.5. 

In essence, it's the very reason why some nugget packages fail to install... The target version for the assembly and the project after 2 different versions, and Unity (or Mono, IDK which) doesn't like that.

**Comment by PrinceRising** [comment]
Yeah i've got the same issue

**Comment by thesteamguy8** [comment]
Hey guys, I have come up with a solution,
        Problem:-                             
                        The problem here is that we do not have the database control or we do not have the paid version of database control, Brackeys did all of his work perfectly, the problem here is that we have the free version and all that stuff.
        Solution:-
                        Now, first of all, if you do not have Database Control, go to your imported project in unity and delete the included database control file provided with the project, after you have done that, Download DatabaseControlFree from here[thats the same file Brackey's used, but an updated version...] :- [https://www.assetstore.unity3d.com/en/#!/content/41337](url) and you are done with the downloading part.
Now in the console,double click on the first script, and it will take you to VisualStudio/MonoDevelop [Depending on what you use, in my case, MonoDevelop], once it opens, you will see the script and replace the Highlighted word [i.e. DC or DatabseControl, followed by a dot(.)  and a word][replace only the DC or DatabaseControl] with 'DCF' go ahead and hit save, and you will notice that the first error will go off. Do the same to all the errors and you are done!!

:+1: If this helped you :100: ,
Support me on patreon : [https://www.patreon.com/TheSteamGuy](url)
Subscribe me on youtube : [https://www.youtube.com/channel/UC8JfEe-OZzqReISIpwX2DQw](url)

If it did not work :-1: ,then quote this reply and let me know.....

~TheSteamGuy
Developer&Hardware&Sotware Engineer

**Comment by nisaldilshan** [comment]
Same Problem Here

**Comment by leonoe** [comment]
sorry to say that, but the link doesnt work for me to download the new database control

**Comment by leonoe** [comment]
OK now it works, thank allot TheSteamGuy! :)

**Comment by leonoe** [comment]
But how can I fix the problem, that no 3d models could be loaded? @thesteamguy8

**Comment by leonoe** [comment]
![sdd](https://cloud.githubusercontent.com/assets/28008408/25404957/f92e2ff8-2a01-11e7-84f9-4848c876225d.PNG)

But how can I fix this problem @thesteamguy8 ???? PLEASE HELP

**Comment by thesteamguy8** [comment]
can you show me the screenshot of the code [the part where the problem occours], also, it would be great if you provide a snippet of the whole code along with the screenshot...

~TheSteamGuy

**Comment by Matspyder51** [comment]
Same problem here : 
![image](https://user-images.githubusercontent.com/1222143/27008241-d0abfecc-4e6c-11e7-9ce6-4cdb03859ad7.png)

![image](https://user-images.githubusercontent.com/1222143/27008243-dcb3f5da-4e6c-11e7-9259-39eb05f38966.png)


![image](https://user-images.githubusercontent.com/1222143/27008244-ea2eb18c-4e6c-11e7-9e56-4d7d8fc976d1.png)

**Comment by thesteamguy8** [comment]
Fix for:
[Null Reference Exception: Object reference not det to an instance of an object.
LoginMenu+<SendRegisterRequest>c__Iterator1.MoveNect () (at Assets/Scripts.....]

who so ever is getting this problem, make sure that you have enabled unity services[multiplayer services], this might be a problem because in the new version of unity, you have to enable the unity multiplayer services to enable internet matchmaking features(this feature has a limit of 10 people).

By enabling unity multiplayer options(as brackeys did), you unlock the feature of online matchmaking options, in the basic version of unity, you are only allowed to invite only 10 players. In the pro version of unity, you are allowed to invite 100players. 

Possibilities of this problem:
1.You have not enabled unity services
2.You are not logged in to your unity account
3.There is a bug in the installed version
4.It is not compatible with the following version

possibilites are endless

There are two solutions to this problem.

1. Install the older version of unity, the best one is the one used by brackeys in the tutorial.
2. Set this Project to LAN only by using unet(Offline) or photon unity(offline).

This is the best solution that I can provide with the given info.

👍 If this helped you 💯 ,
Support me on patreon : https://www.patreon.com/pranesharry?alert=1
Subscribe me on youtube : https://www.youtube.com/channel/UC8JfEe-OZzqReISIpwX2DQw

If it did not work 👎 ,then quote this reply and let me know.....

~TheSteamGuy

**Comment by thesteamguy8** [comment]
Fix For
[Null Reference Exception: Object reference not det to an instance of an object.
LoginMenu.Start() (At Assets/....]

This is a simple fix, you may not have set the menu properly.
For more info check the Brackey's video on menu's! 

👍 If this helped you 💯 ,
Support me on patreon : https://www.patreon.com/pranesharry?alert=1
Subscribe me on youtube : https://www.youtube.com/channel/UC8JfEe-OZzqReISIpwX2DQw

If it did not work 👎 ,then quote this reply and let me know.....

~TheSteamGuy

**Comment by burakmert61** [comment]
![error](https://user-images.githubusercontent.com/30290937/28366579-adb09948-6c8d-11e7-8289-a5b00178ecbf.png)

**Comment by enderGames** [comment]
> Hey guys, I have come up with a solution,
Problem:-
The problem here is that we do not have the database control or we do not have the paid version of database control, Brackeys did all of his work perfectly, the problem here is that we have the free version and all that stuff.
Solution:-
Now, first of all, if you do not have Database Control, go to your imported project in unity and delete the included database control file provided with the project, after you have done that, Download DatabaseControlFree from here[thats the same file Brackey's used, but an updated version...] :- https://www.assetstore.unity3d.com/en/#!/content/41337 and you are done with the downloading part.
Now in the console,double click on the first script, and it will take you to VisualStudio/MonoDevelop [Depending on what you use, in my case, MonoDevelop], once it opens, you will see the script and replace the Highlighted word [i.e. DC or DatabseControl, followed by a dot(.) and a word][replace only the DC or DatabaseControl] with 'DCF' go ahead and hit save, and you will notice that the first error will go off. Do the same to all the errors and you are done!!
👍 If this helped you 💯 ,
Support me on patreon : https://www.patreon.com/TheSteamGuy
Subscribe me on youtube : https://www.youtube.com/channel/UC8JfEe-OZzqReISIpwX2DQw
If it did not work 👎 ,then quote this reply and let me know.....
~TheSteamGuy
Developer&Hardware&Sotware Engineer

Hi, I'm trying to put the script called "dcDemo_MenuControl_CSharp.cs" on the MenuControl in the inspector, but I can't. Unity says: "no monobehaviour scripts in the file". The filename and class name are the same, and there are no errors or warnings in the file. Help?

**Comment by enderGames** [comment]
@thesteamguy8 ^

**Comment by 0xhex** [comment]
![screenshot 2017-10-29 02 51 34](https://user-images.githubusercontent.com/7974431/32139534-1bb5173c-bc54-11e7-8e90-f93bb6c0be47.png)

any idea?login and register not working,when i try to login with lobby scene,player is on the air

**Comment by Arjay01** [comment]
@thesteamguy8 

https://www.assetstore.unity3d.com/en/#!/content/41337 

^^^ link does not work, just says 404 content not found.
Is there any other link?

**Comment by KhachNazar** [comment]
Had the same issues described above. The solution is very easy. The problem is that the developer changed a bunch of things and now you just have to write 
 string returnedd = ee.Current as string;
and not 
WWW returnedd = ee.Current as WWW;
then change everywhere returnedd.text to returnedd. everything c]should work. 
Do it also with "returned"
![screenshot 47](https://user-images.githubusercontent.com/37668752/42092737-079feed0-7bbb-11e8-9907-823c6c468c84.png)

**Comment by kylooes** [comment]
Hey guys,i'm a chinese guy having watched this videos ,and i'm facing a lots bothering issues but I really want to have this project.
I'll utterly appreciate it if some of you send me a entire unity project which is checked and having no problems to buid and play with imformations about unity version used;
my emailadress: 1297044744@qq.com
thanks.

**Comment by kylooes** [comment]
@thesteamguy8

**Comment by Cyroen** [comment]
i have a problem when i login/register it just infinite loads! please help!
